### DISCUSSION TITLE
Point Selection

### DISCUSSION THREAD

**Discussion by hosseinnegahban** [mdolab/dafoam#268] [Tutorials]
Hey guys,

I want to selct specified point for my case but when I select the point for a 2D airfoil case in trailing edge, only one side of the airfoil (z=0 or z=z0) deforms and symmetry constraint does not work at all.  I have modified python code  a lot
![Screenshot 2022-02-18 22 05 43](https://user-images.githubusercontent.com/97816713/154783763-013cd736-00ba-4c3c-875e-aadad2598efb.png)
 but no solution by now. Do you know how to solve it?

**Comment by friedenhe** [comment]
I suspect that you did not add the symmetry constraint. Can you post your run script here?

**Comment by hosseinnegahban** [comment]
Actually, I only have modified the "select Point" section and have not touched constraints, except that I have unconstrained the trailing edge point in TeLe constraint. No modifications in symmetry constraint. Here is my runScript file in ZIP file
[runScript.zip](https://github.com/mdolab/dafoam/files/8110126/runScript.zip)
.

**Comment by friedenhe** [comment]
Your change was not right. Check how Python indexing is working.... Also, check the workshop video on how to set the symmetry constraint.
